kafka-commons/src/test/java/uk/gov/homeoffice/digital/sas/transactionsync/KafkaDbTransactionSynchronizerIntegrationTest.java

```diff
@@ -35,17 +35,21 @@
 )
 class KafkaDbTransactionSynchronizerIntegrationTest {
   private Long profileId;
+  private Long tenantId;
   private static final String PROFILE_NAME = "Original profile";
   private Profile profile;
 
+  private String messageKey;
+
   @Autowired
   private ProfileRepository profileRepository;
 
   @BeforeEach
   void setup() {
     TransactionSynchronizationManager.initSynchronization();
     profileId = new Random().nextLong();
-    profile = new Profile(String.valueOf(profileId), PROFILE_NAME);
+    tenantId = new Random().nextLong();
+    profile = new Profile(String.valueOf(profileId), String.valueOf(tenantId), PROFILE_NAME);
```

Comment:
I think it would be more realistic to not provide the profile Id here as this PR is to be able to get a message key based on the db ID.
So to test that works, realistically we'd want to provide a fresh resource without the ID, let the db assign it and then expect that it gets resolved after that and be present in the log.

Reply:
I have pulled @taomoh suggested changes https://github.com/UKHomeOffice/callisto-jparest/pull/62 in. This should address your comment.